asylum-connect/control-panel
# Add Washington to District of Columbia

## Description

The city of "Washington" was now showing up under the state "District of Columbia" because we were searching for "District Of Columbia" with a capitalized "Of". The solution was to lower case all state names before we look up their cities. 

I also replaced the open and close parenthesis in the "Optional" form label with the equivalent HTML entity tags. This looks the same to user but fixes an issue where Intellij was unable to parse the JSX in this file and thus the syntax highlighting was all messed up. 

- Asana ticket: https://app.asana.com/0/1132189118126148/1189278038283162

## How to Test

Select "District of Columbia" in the coverage editor, then verify that "Washginton" appears in the city dropdown.
